app/assets/javascripts/discourse/app/templates/components/group-flair-inputs.hbs

```diff
@@ -21,11 +21,12 @@
       onChange=(action (mut model.flair_icon))
     }}
   {{else if flairPreviewImage}}
-    {{image-uploader
+    {{uppy-image-uploader
       imageUrl=flairImageUrl
       onUploadDone=(action "setFlairImage")
       onUploadDeleted=(action "removeFlairImage")
       type="group_flair"
+      id="group-flair-uploader"
```

Comment:
Hmm I wonder if we should have a consistent ID story for the uploaders.

1. `category-background-upload`
2. `group-flair-uploader`
3. `profile-background`

The IDs are named very differently between components.